### PR TITLE
[WIP] Add eus_vive package

### PR DESCRIPTION
**Pull request by knorth55**
I use this ROS package to run Miraikan  openlab demo in 2019.
How to run the demo is written here https://github.com/knorth55/jsk_control/blob/4732ce5b3f76259c2c809714e16c539c47134c06/eus_vive/README.md

- [x] Vive
- [x] Oculus (Partially)
- [x] Spacenav
- [ ] Tablis

TODO: modify to use https://github.com/jsk-ros-pkg/jsk_robot/pull/1541

### CODE REVIEW

**Review by k-okada — CHANGES_REQUESTED** [review]

File: eus_vive/kinetic.rosinstall [hunk 1]
```diff
@@ -0,0 +1,16 @@
+- git:
```
[the remark this answers is not in the document]
Comment (knorth55) [reply eus_vive/kinetic.rosinstall]: I updated to add information

File: eus_vive/package.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>eus_vive</name>
+  <version>0.0.0</version>
```
[the remark this answers is not in the document]
Comment (knorth55) [reply eus_vive/package.xml]: I update this package version.

**Comment by knorth55** [comment]
Now i update this PR and support spacenav_node.
`spacenav_node` publish `Joy` topic, so I can support other joy devices, too.
`omega_devices`  are a bit different because it has feedback, so i need update this package more.

**Comment by k-okada** [comment]
> so i need update this package more.

OK, added `[WIP]` in the title.

**Pull request closed without merging**